packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -40,4 +42,19 @@ AppConfig.restore = () => {}
 AppConfig.extraGetPropsArgs = () => {}
 AppConfig.freeze = () => {}
 
-export default AppConfig
+// Recommended settings for PWA-Kit usage.
+const options = {
+    queryClientConfig: {
+        defaultOptions: {
+            refetchOnMount: false,
```

Comment:
I believe this is because by default React Query considers all queries as stale with `staleTime: 0`.
https://tanstack.com/query/v4/docs/guides/important-defaults#:~:text=by%20default%20consider%20cached%20data%20as%20stale

You can try increasing the staleTime in the client options: 
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/84fac33699bcf85cf6c21665fa88527b43ad4230/packages/test-commerce-sdk-react/app/components/_app-config/index.tsx#L45-L58

```
// Recommended settings for PWA-Kit usage.
const options = {
    queryClientConfig: {
        defaultOptions: {
            refetchOnMount: false,
            queries: {
                retry: false,
                staleTime: 1000 * 20 // 20 seconds. Default value is 0
            },
            mutations: {
                retry: false
            }
        }
    }
}
```

Reply:
@bendvc Can you explain why we need `refetchOnMount: false` for all queries?

Also note that the global configuration applied server-side and client-side is not going to work for everyone, for example when users need to work with a flaky APIs they would likely want to increase the number of retries adding a delay in the rendered content to users vs showing users an error and not allowing them to proceed.

```
queries: {
    retry: false,
    staleTime: 1000 * 20 // 20 seconds. Default value is 0
},
mutations: {
    retry: false
}
```

Reply:
Sorry guys.. I have removed that as I was experimenting with ways to stop queries running during hydration.